packages/workbox-google-analytics/src/index.js

```diff
@@ -19,12 +19,10 @@ import logHelper from '../../../lib/log-helper.js';
 import replayQueuedRequests from './lib/replay-queued-requests.js';
 
 /**
- * In order to use the library, call
- * `workbox.googleAnalytics.initialize()`.
- * It will take care of setting up service worker `fetch` handlers to ensure
- * that the Google Analytics JavaScript is available offline, and that any
- * Google Analytics requests made while offline are saved (using `IndexedDB`)
- * and retried the next time the service worker starts up.
+ * A module that ensures Google Analytics works offline. It does this by
```

Comment:
Would this not be a library? I see `module` as being used for packages that can really only be consumed via Node while browser/client facing ones are a library an end user consumes more generally.

`workbox-precaching` would be a module.